src/stores/SSEChannel/MessagesSSEChannel.ts

```diff
@@ -153,7 +155,30 @@ export class MessagesSSEChannel extends SSEChannel<EventMessage> {
 	private _onMessageIdsEvent = (e: Event) => {
 		const messagesIdsEvent: MessageIdsEvent =
 			e instanceof MessageEvent && e.data ? JSON.parse(e.data) : null;
-		this.messageIds = Object.values(messagesIdsEvent.messageIds).filter(Boolean) as string[];
+		const streams = getArrayOfUniques(
+			this.messageIds
+				.filter(id => id.slice(id.lastIndexOf(':') + 1) === '-1')
```

Comment:
`id.slice(-2) == '-1'` looks more readable i think?
btw if we extract it to helper functions it would be even more readable
`this.messageIds.filter(isInvalidResumeId).map(getSession)`